instaloader/nodeiterator.py

```diff
@@ -77,9 +79,11 @@ def __init__(self,
                  query_variables: Optional[Dict[str, Any]] = None,
                  query_referer: Optional[str] = None,
                  first_data: Optional[Dict[str, Any]] = None,
-                 is_first: Optional[Callable[[T, Optional[T]], bool]] = None):
+                 is_first: Optional[Callable[[T, Optional[T]], bool]] = None,
+                 doc_id: Optional[str] = None):
```

Comment:
I would appreciate a `versionadded` comment in the docstring that briefly describes this change.